lib/engine/game/g_1847_ae/game.rb

```diff
@@ -211,10 +243,17 @@ def init_corporations(stock_market)
         end
 
         def setup
+          # Place L's home station in case there is a "short OR" during draft
+          hex = hex_by_id(l.coordinates)
+          tile = hex&.tile
```

Comment:
Could `hex` be nil here? If it is, then `tile` will be set to nil, and the next line will throw an error.
I can't see why the call to `hex_by_id` would return nil, so the `&` should be removed.